Medium/Mirror.js

```diff
@@ -0,0 +1,15 @@
+//Given an integer array, transform that array into a mirror
+//example mirror([1, 2, 3, 4, 5]) ➞ [1, 2, 3, 4, 5, 4, 3, 2, 1]
+
+const mirror=(arr)=>
+{
+
+    let mirrorarr=[];
+    for (let i = 0; i < arr.length; i++) {
```

Comment:
No need to create this loop, you can concate mirror array with this
https://www.w3schools.com/jsref/jsref_concat_array.asp